main.rb

```diff
@@ -0,0 +1,162 @@
+require_relative './student'
+require_relative './classroom'
+require_relative './person'
+require_relative './rentals'
+require_relative './teacher'
+require_relative './book'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      return
+    end
+
+    print 'Age: '
```

Comment:
- [ ] the role of main.rb is only to start the code and ask the user for the initial choice of options (or to quit). This code should be in another file, for example, app.rb